scripts/asbcli/asbcli

```diff
@@ -460,14 +468,16 @@ class App(object):
 # 24   -v \
 # 25   http://cap.example.com:1338/v2/service_instances/$instanceUUID
 ############################################################
-    def _req_provision(self, service_id, answers):
+    def _req_provision(self, service_id, answers, namespace):
```

Comment:
👍 

I think we also need this for bind, unbind, and deprovision.  Any place that launches an apb container will require a namespace.